var/spack/repos/builtin/packages/rust-bootstrap/package.py

```diff
@@ -133,6 +133,12 @@ class RustBootstrap(Package):
         if os in rust_releases[release] and target in rust_releases[release][os]:
             version(release, sha256=rust_releases[release][os][target])
 
+    # rust-ldd and libLLVM both depend on zlib, which is not vendored.
+    depends_on("zlib-api")
```

Comment:
Can variants could be passed through the virtual? Or was that only the version?

Reply:
neither

Reply:
By passthrough I should be more clear. Can you define variants on a virtual package. I know you can define versions, I did that for `libllvm` to translate ROCm versions to `llvm` versions for the virtual.

If so, it would maybe be more concise to do that here.

Reply:
Nope, maybe in a future Spack.